docs/development.md

```diff
@@ -5,26 +5,13 @@
 Create a GCP environment.
 
 ```sh
-make install
+make dev-up
 ```
 
-Setup controller for running locally.
+Run Substratus control plane locally.
 
 ```sh
-# Example only: use your own script.
-source ./hack/dev/example-gcp-env.sh
```

Comment:
I think we still need to source our own script? so we should probably keep this in?

Reply:
nvm I see this is now done in the Makefile. Nice!